server/neptune/workflows/internal/main.go

```diff
@@ -0,0 +1 @@
+package internal
```

Comment:
?

Reply:
Go complained that there wasn't a go file within the internal package directly, so I just quickly threw in a main.go file to silence the error.